go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/apache/airflow-client-go
+
+go 1.13
```

Comment:
On CI we use `1.15` should we upgrade also here?

Reply:
we would want to keep this version as low as possible for ease of adoption. I will add a new job in CI to test run 1.13 as well.